internal/runners/push/push.go

```diff
@@ -162,7 +162,28 @@ func (r *Push) Run(params PushParams) error {
 	// update the project at the given commit id.
 	err = model.UpdateProjectBranchCommitWithModel(pjm, branch.Label, commitID)
 	if err != nil {
-		return locale.WrapError(err, "push_project_branch_commit_err", "Failed to update new project {{.V0}} to current commitID.", pjm.Name)
+		if errs.Matches(err, &model.ErrUpdateBranchAuth{}) {
```

Comment:
Let's invert this check and return early, then this whole block doesn't need to be indented.